test/sign_blob_test.sh

```diff
@@ -30,7 +30,7 @@ date > $BLOB
 cat $BLOB
 
 echo "Sign the blob with cosign first and upload to rekor"
-$COSIGN_CLI sign-blob --output-certificate blob.cert --output-signature blob.sig $BLOB
+$COSIGN_CLI sign-blob -y --output-certificate blob.cert --output-signature blob.sig $BLOB
```

Comment:
Not: Can we make that explicity with the full flag name?

Reply:
yes! done